app/pages/forms/formdemo.js

```diff
@@ -0,0 +1,28 @@
+import Head from "next/head";
+import RCform from "../../components/clientForms/show";
+import { getForms } from "../../lib/formAPI";
+
+function FormDemo({ formFields }) {
+  return (
+    <div>
+      <Head>
+        <title>Form</title>
+        <meta name="description" content="Rocket.Chat form tool demo" />
+        <link rel="icon" href="/favicon.ico" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      </Head>
+      <RCform formFields={formFields} />
```

Comment:
Need to craft `RCform` as a re-usable component.   

Show us on this demo page.   Add 4 or 5  instances of different forms.

Similar to:
```
<h1>Sample of Form Component in Action</h1>
<Row>
<Col>
  <RCform  id="customersatisfaction" />
  </Col>
  <Col>
   <RCform id="survey1"/>
   </Col>
 </Row>
 <Row>
<Col>
  <RCform  id="mostlovedmusic" />
  </Col>
  <Col>
   <RCform id="COVID19Questionnaire"/>
   </Col>
 </Row>
 ```